assets/pages/digital-subscription-checkout/components/thankYouContent.jsx

```diff
@@ -83,18 +92,22 @@ function EmailsSection() {
 
   return (
     <LeftMarginSection modifierClasses={['thank-you-emails']}>
-      <h2 className="thank-you-content__heading">
-        Stay in the know
-      </h2>
-      <p className="thank-you-content__copy">
-        <strong className="thank-you-content__strong">
-          Opt in below to receive news and offers from The Guardian, The Observer and Guardian Weekly
-        </strong> on the ways to read and support our journalism. Already a member,
-        subscriber or contributor, opt in here to receive your regular emails and updates.
-      </p>
-      <p className="thank-you-content__copy thank-you-content__copy--small">
-        <small>You can unsubscribe at any time by going to the preference centre</small>
-      </p>
+      <div className="thank-you-content__email-copy">
+        <h2 className="thank-you-content__heading">
+          Stay in the know
+        </h2>
+        <p className="thank-you-content__copy">
+          Opt in below to receive news and offers from The Guardian, The Observer
+          and Guardian Weekly on the ways to read and support our journalism.
+        </p>
+        <p className="thank-you-content__copy thank-you-content__copy--small">
+          <small>
+            <SvgInformation />
+            You can unsubscribe at any time by going to the <a className="thank-you-content__link" href={emailPreferencesUrl}>preference centre</a>
```

Comment:
Can we add `rel="noreferrer noopener"` - https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
Ah, yes. And as we're not setting `target` here, please ignore me 😄